### PR TITLE
Barbican api check

### PR DESCRIPTION
**Pull request by wupeiran**
Add barbican api check for monitoring

### CODE REVIEW

File: sensu/plugins/check-os-api.rb [hunk 1]
```diff
@@ -35,6 +35,9 @@ def safe_command
     when "ceilometer"
       "ceilometer meter-list -l 1"
     end
```
Comment (Xiaohua-Shen) [inline sensu/plugins/check-os-api.rb]: should omit this 'end'.

**Review by Xiaohua-Shen — CHANGES_REQUESTED** [review]

File: sensu/plugins/check-os-api.rb (hunk 1 above)
Comment (wupeiran) [reply sensu/plugins/check-os-api.rb]: miss the last end. will update

**Comment by nirajdp76** [comment]
We should make the list calls light weight. Goes not all services not just barbican. 
Look at each command help and see what options it provides. --limit 1, etc ..

**Review by nirajdp76 — CHANGES_REQUESTED** [review]
see comment

**Comment by wupeiran** [comment]
update commands for list calls light weight

**Review by nirajdp76 — APPROVED** [review]

**Pull request merged**